zulipterminal/ui_tools/buttons.py

```diff
@@ -15,25 +15,31 @@
 from zulipterminal.urwid_types import urwid_Size
 
 
+urwidMarkupTuple = Tuple[Optional[str], str]
+
+
 class TopButton(urwid.Button):
     def __init__(
         self,
         *,
         controller: Any,
-        caption: str,
+        prefix_markup: urwidMarkupTuple = (None, ""),
+        label_markup: urwidMarkupTuple,
+        suffix_markup: urwidMarkupTuple = (None, ""),
```

Comment:
I feel markup is a better description here as it the "style wrapper for any text" from what I understand reading the urwid code. The problem with using `styled_prefix` for eg. is it might get confused with `prefix_style` introduced in the later commits.

Reply:
Also we use markup in many places in the code.

Reply:
OK, I'm convinced based on what the rest of the code does.
The problem with referring to the urwid code is that we don't always use the same terms, eg. we use `styles`.